### DISCUSSION TITLE
Could `crossrefs-hover: true` work for websites?

### DISCUSSION THREAD

**Discussion by olivroy** [quarto-dev/quarto-cli#9641] [Feature Requests]
### Description

I think from what I understand, cross-refs are only enabled for documents, manuscripts and book projects.

In websites, cross-references are not supported and references should be used via [linking](https://quarto.org/docs/websites/#linking).

So I am wondering if these links could listen to `crossref-hover`. So that it would be possible to preview a section or a figure from another page in a Quarto website.

In short, I'd love to be able to preview sections or figures like you see in a book, but on a website. I could see some benefit of this feature on the Quarto.org website also.
